packages/frontend/core/src/pages/workspace/tag.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  currentCollectionAtom,
+  TagListHeader,
+  useTagMetas,
+} from '@affine/core/components/page-list';
+import { useBlockSuitePageMeta } from '@affine/core/hooks/use-block-suite-page-meta';
+import { waitForCurrentWorkspaceAtom } from '@affine/core/modules/workspace';
+import { getCurrentStore } from '@toeverything/infra/atom';
+import { useAtomValue } from 'jotai';
+import { useMemo } from 'react';
+import { type LoaderFunction, redirect, useParams } from 'react-router-dom';
+import { NIL } from 'uuid';
+
+import { AllPage } from './all-page/all-page';
+import { AllPageHeader } from './all-page/all-page-header';
+import { EmptyPageList } from './page-list-empty';
+
+export const loader: LoaderFunction = async args => {
+  const rootStore = getCurrentStore();
+  if (!args.params.tagId) {
+    return redirect('/404');
+  }
+  rootStore.set(currentCollectionAtom, NIL);
```

Comment:
Is this required for this route?

Reply:
Originally it was a shared filteredPageMetas, so the initial value needed to be restored, but now it is no longer needed and I have deleted it.